index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sam's Personal Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
+</head>
+<body>
+    <div class="header">
+        <nav class="nav">
+            <div class="myLogo">
+                My Logo
+            </div>
+            <div class="sideNav">
+                <img src="images/Union.png" alt="Side navigation bar icon">
+            </div>
+        </nav>               
+    </div>
+    <div class="headline">
+        <div class="textbx">
+               <div>
+                <h1>
+                    I’m Sam <br> Glad to see you!
+                </h1>
+                <p>
+                    I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                </div>
+                <div class="connect">
+                    LET'S CONNECT
+                </div>
+                <div class="mediaIcons">
+                    <div class="twitter">
+                        <a href="#">
+                            <img src="images/twitter.png" alt="twitter icon">
+                         </a>
+                        </div>
+                    <div class="linkedIn">
+                        <a href="#">
+                            <img src="images/Linkedin.png" alt="LinkedIn icon">                        
+                        </a>
+                    </div>
+                    <div class="medium">
+                        <a href="#">
+                            <img src="images/medium.png" alt="Medium Icon">
+                        </a>
+                    </div>
+                    <div class="github">
+                    <a href="#">
+                        <img src="images/github.png" alt="Github Icon">
+                    </a>
+                    </div>
+                    <div class="angellist">
+                    <a href="#">
+                    <img src="images/angellist.png" alt="Angellist icon">
+                    </a>
+                    </div>
+                </div>                   
+                
+            </div>
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon should have a link (`a` tag).